ansible.cfg

```diff
@@ -1 +1,2 @@
-[defaults]
\ No newline at end of file
+[defaults]
+callback_whitelist = profile_tasks
```

Comment:
Happy to remove it, I thought it could be useful for historical comparison, and I believe it's only use when the test suite is run. It won't be used by anyone consuming the role right?

Reply:
Yeah also it's really useful, we don't currently use it and I would prefer removing it of the repository even if there isn't any impact for users consuming the role.

Reply:
Removed!